src/components/v2/V2Project/index.tsx

```diff
@@ -46,8 +39,8 @@ export default function V2Project() {
         <Col md={12} xs={24}>
           <PayInputGroup
             PayButton={V2PayButton}
-            reservedRate={reservedRatePercent}
-            weight={weight}
+            reservedRate={fundingCycleMetadata?.reservedRate.toNumber()}
```

Comment:
smol fix, this takes a permyriad not a percent

Reply:
Nice